src/App.jsx

```diff
@@ -0,0 +1,22 @@
+import { useState } from "react";
+import { MainPage } from "./pages/MainPage";
+import { ProfilePage } from "./pages/ProfilePage";
+import { Navbar } from "./components/Navbar";
+
+function App() {
+  const pages = {
+    home: "Home",
+    profile: "Profile",
+  };
+
+  const [currentPage, setCurrentPage] = useState(pages.home);
+
+  return (
+    <div className="max-w-screen-xl mx-auto px-4 sm:px-6 lg:px-8 text-center">
```

Comment:
The page will look better if you don't restrict the width of the navbar.
Try this:
```JSX
    <>
      <Navbar setCurrentPage={setCurrentPage} pages={pages} />
      <div className='max-w-screen-xl mx-auto p-4 sm:p-6 lg:px-8 text-center'>
        {currentPage === pages.home && <MainPage />}
        {currentPage === pages.profile && <ProfilePage />}
      </div>
    </>
```